app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -532,10 +532,6 @@ export default Component.extend(TextareaTextManipulation, {
       },
 
       onKeyUp: (text, cp) => {
-        if (inCodeBlock(text, cp)) {
```

Comment:
Not sure what this `onKeyUp` function is doing 🤔 I removed it and couldn't see any difference in behaviour locally.